src/Tooltip/examples/positions.js

```diff
@@ -0,0 +1,44 @@
+class Example extends React.Component {
```

Comment:
evaluating with @alessandravilla... consider that we will have noise due to the dropdowns logic.